src/_P064_APDS9960.ino

```diff
@@ -22,18 +27,19 @@
 #define PLUGIN_VALUENAME1_064 "Gesture"
 #define PLUGIN_VALUENAME2_064 "Proximity"
 #define PLUGIN_VALUENAME3_064 "Light"
-/*
+
 #define PLUGIN_VALUENAME4_064 "R"
 #define PLUGIN_VALUENAME5_064 "G"
 #define PLUGIN_VALUENAME6_064 "B"
-*/
+

```

Comment:
These value names reflect 2 sets of 3, so maybe better to let the name of the define also be more descriptive?

For example: 
- `P064_MOTION_VALUENAME1` ... `3` 
- `P064_COLOR_VALUENAME1` ... `3`

Reply:
Done.